plans/provision.pp

```diff
@@ -44,6 +62,7 @@
   $tfvars = inline_epp(@(TFVARS))
     project         = "<%= $project %>"
     user            = "<%= $ssh_user %>"
+    lb_ip_mode      = "<%= $lb_ip_mode %>"
```

Comment:
Can we add `ssh_ip_mode` here as well? We can pass it through to the terraform instances module and switch all of the conditionals to leverage it. 

https://github.com/puppetlabs/terraform-google-pe_arch/blob/main/modules/instances/main.tf#L35
https://github.com/puppetlabs/terraform-google-pe_arch/blob/main/modules/instances/main.tf#L76
https://github.com/puppetlabs/terraform-google-pe_arch/blob/main/modules/instances/main.tf#L113
https://github.com/puppetlabs/terraform-google-pe_arch/blob/main/modules/instances/main.tf#L149

Reply:
It is probably a bad parameter name for that use case, `instance_ip_mode` maybe is better?

Reply:
That will be good and we can use it within the terraform code to determine the configuration. I like the idea of the `mode` over a boolean since it has the ability to add additional configuration later without adding another parameter.